### PR TITLE
LPS-159977 Add tests for useSessionState

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:test:relevant

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/core/hooks/useSessionState.test.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ * details.
+ */
+
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import {useSessionState} from '../../../../src/main/resources/META-INF/resources/page_editor/core/hooks/useSessionState';
+
+const renderHook = ({defaultValue = null, key}) => {
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/test/page_editor/core/hooks/useSessionState.test.js:21]: You can use `import {renderHook} from '@testing-library/react-hooks';` for this purpose, did you try it?

**Comment by victorg1991** [comment]
ci:forward

**Comment by p2kmgcl** [comment]
ci:stop

**Comment by p2kmgcl** [comment]
ci:forward

**Pull request closed without merging**